_Week5/group4/program.do

```diff
@@ -0,0 +1,27 @@
+cap prog drop graphsum

```

Comment:
I think it would be worth adding to the readme file that any previous programs called graphsum the user may have installed will be deleted.